pkg/validation/validate_cluster.go

```diff
@@ -212,17 +210,32 @@ func (v *ValidationCluster) collectPodFailures(client kubernetes.Interface, node
 		}
 	}
 
-	for _, pod := range pods.Items {
+	err := pager.New(pager.SimplePageFunc(func(opts metav1.ListOptions) (runtime.Object, error) {
+		return client.CoreV1().Pods(metav1.NamespaceAll).List(opts)
+	})).EachListItem(context.TODO(), metav1.ListOptions{}, func(obj runtime.Object) error {
+		pod := obj.(*v1.Pod)
+		priority := pod.Spec.PriorityClassName
+		if priority != "system-cluster-critical" && priority != "system-node-critical" {
```

Comment:
Review note (no change needed): These used to be permitted only in kube-system, but as of https://github.com/kubernetes/kubernetes/pull/76310 (i.e. k8s 1.17) that is no longer the case.